topics/imaging/tutorials/yolo_prediction/tutorial.md

```diff
@@ -0,0 +1,282 @@
+
+contributions:
+  authorship:
+    - mjoudy
+    - yvanlebras
+
+follow_up_training:
+  - type: "internal"
+    topic_name: machine-learning
+    tutorials:
+      - ml-advanced-image
+
+---
+
+# Introduction
+
+YOLO (You Only Look Once) is a fast, deep learning-based algorithm for real-time object detection. It predicts object classes and bounding boxes in a single pass over the image. YOLOv8 is a specific version, offering improved accuracy and speed.
+
+In this tutorial, you will use Galaxy to run two types of YOLOv8 models:
+
+1. **Object Detection** using real underwater images from the SEANOE dataset.
+2. **Segmentation** using an Ultralytics demo model on a standard image.
+
+We'll compare both modes and discuss what kind of output they generate and why it matters in bioimage analysis.
+
+---
+
+# Part 1: Detection of marine species
+
+## 🔗 Dataset
+
+We will use selected images from the SEANOE dataset:
+
+👉 [SEANOE 101899 - Deep-sea marine species observations](https://www.seanoe.org/data/00907/101899) (Lebeaud *et al.*, 2024 [https://doi.org/10.17882/101899](https://doi.org/10.17882/101899))
+
+The SEANOE #101899 collection features real underwater images captured by deep‑sea observatories as part of a citizen science initiative called Deep Sea Spy. These non‑destructive imaging stations continuously monitor marine ecosystems and provide snapshots of various fauna. In this dataset, multiple annotators—including trained scientists and enthusiastic citizen scientists—have manually labeled images with polygons, lines, or points highlighting marine organisms. These annotations were then cleaned and converted into bounding boxes to create a training-ready dataset for object detection with YOLOv8. Though the exact species vary, images often include deep-sea fish, species, making this dataset well-suited for practicing detection tasks.
+
+## Get data
+
+> <hands-on-title> Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial and give it a name (example: “DeepSeaSpy Yolo tutorial”) for you to find it again later if needed.
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+>    {% snippet faqs/galaxy/histories_rename.md %}
+>
+> 2. Import images data files and models from [SEANOE marine datawarehouse](https://www.seanoe.org/data/00907/101899/).
+>
+>    DeepSeaSpy images data files and models as a zip file
+>    ```
+>    https://www.seanoe.org/data/00907/101899/data/115473.zip
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+> 3. Use  {% tool [Unzip a file](toolshed.g2.bx.psu.edu/repos/imgteam/unzip/unzip/6.0+galaxy0) %} to create a data collection in your history where all archive files will be unzipped.
+>
+> 4. Unhide the models data files.
+>
+>    History search `name:detection deleted:false visible:any` then unhidde the 2 model files "dataset_seanoe_101899_YOLOv8-weights-for-Bythograeidae-detection" and "dataset_seanoe_101899_YOLOv8-weights-for-Buccinidae-detection".
```

Comment:
```suggestion
>    History search `name:detection deleted:false visible:any` then show the 2 model files "dataset_seanoe_101899_YOLOv8-weights-for-Bythograeidae-detection" and "dataset_seanoe_101899_YOLOv8-weights-for-Buccinidae-detection".
```

Reply:
It's actually called "unhide" in Galaxy:

<img width="289" height="168" alt="Image" src="https://github.com/user-attachments/assets/0bc91154-9aa5-473f-b6d1-047cb40a5f73" />